src/log-manager/config/log-manager.yaml

```diff
@@ -18,3 +18,7 @@
 service_type: "k8s"
 
 port: 9103
+admin_name: "admin"
```

Comment:
Please add this change to https://github.com/microsoft/pai/blob/master/contrib/kubespray/quick-start/services-configuration.yaml.template#L239

Reply:
Fixed